### PR TITLE
Add @adriaanm-da to the release rotation

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
@adriaanm-da this should put you in charge of the snapshot release in the first week of October.

changelog_begin
changelog_end

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by cocreature — APPROVED** [review]
Assuming @adriaanm-da is fine with that, that sounds like a good idea :)

@adriaanm-da Our process is documented in https://github.com/digital-asset/daml/blob/main/release/RELEASE.md but ofc if anything is unclear feel free to ask (and ideally improve the docs afterwards).

**Comment by stefanobaghino-da** [comment]
> Assuming @adriaanm-da is fine with that, that sounds like a good idea :)
> 
> @adriaanm-da Our process is documented in https://github.com/digital-asset/daml/blob/main/release/RELEASE.md but ofc if anything is unclear feel free to ask (and ideally improve the docs afterwards).

I suggested the idea yesterday and he agreed, I'll wait for his approval to make sure the timing fits his schedule.

**Comment by stefanobaghino-da** [comment]
He approved off-band. Merging. /cc @adriaanm-da

**Pull request merged**
